### DISCUSSION TITLE
英文封面的学位

### DISCUSSION THREAD

**Discussion by Forest216** [nju-lug/NJUThesis#200] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/41437440/232368388-15362e55-c926-47d2-9458-21c797dc4c5c.png)
![image](https://user-images.githubusercontent.com/41437440/232368437-8572fe64-7fb9-458a-a872-f70dd311c571.png)
请问如何将如图一中的MASTER设置为如图二的MASTER OF ENGINEERING即工程硕士

**Comment by atxy-blip** [comment, marked as answer]
导言区加如下代码
```tex
\ExplSyntaxOn
\tl_set:Nn \g__nju_info_type_en_tl { Master~ of~ Engineering }
\ExplSyntaxOff
```
考虑之后出个设置项简化一下。

**Reply by Forest216** [reply]
感谢

**Reply by atxy-blip** [reply]
我在 <https://github.com/nju-lug/NJUThesis/commit/29402a5a3f6133409b5542b17bd98cd13695e608> 加了一个选项 `info/degree` 来实现你的需求，随后发布的 v1.2 版和这个补丁是不兼容的。

**Reply by Forest216** [reply]
赞
